interface/patient_tracker/patient_tracker.php

```diff
@@ -10,71 +10,71 @@
+$facility = prevSetting($uspfx, 'form_facility', 'form_facility', '');
+$provider = prevSetting($uspfx, 'form_provider', 'form_provider', $_SESSION['authUserID']);
 
-if (!is_null($_POST['form_provider'])) {
-    $provider = $_POST['form_provider'];
-} else if ($_SESSION['userauthorized']) {
-    $provider = $_SESSION['authUserID'];
-} else {
-    $provider = null;
+if (($_POST['setting_new_window']) ||
+    ($_POST['setting_bootstrap_submenu']) ||
+    ($_POST['setting_selectors'])) {
+    // These are not form elements. We only ever change them via ajax, so exit now.
+    exit();
+}
+if ($_POST['saveCALLback'] == "Save") {
+    $sqlINSERT = "INSERT INTO medex_outgoing (msg_pc_eid,msg_pid,campaign_uid,msg_type,msg_reply,msg_extra_text)
+                  VALUES
+                (?,?,?,'NOTES','CALLED',?)";
+    sqlQuery($sqlINSERT, array($_POST['pc_eid'], $_POST['pc_pid'], $_POST['campaign_uid'], $_POST['txtCALLback']));
 }
+$from_date = !is_null($_REQUEST['datepicker1']) ? date('Y-m-d', strtotime($_REQUEST['datepicker1'])) : date('Y-m-d');
 
-$facility  = !is_null($_POST['form_facility']) ? $_POST['form_facility'] : null;
-$form_apptstatus = !is_null($_POST['form_apptstatus']) ? $_POST['form_apptstatus'] : null;
-$form_apptcat=null;
-if (isset($_POST['form_apptcat'])) {
-    if ($form_apptcat!="ALL") {
-        $form_apptcat=intval($_POST['form_apptcat']);
-    }
+
+if (substr($GLOBALS['ptkr_end_date'], 0, 1) == 'Y') {
+    $ptkr_time = substr($GLOBALS['ptkr_end_date'], 1, 1);
+    $ptkr_future_time = mktime(0, 0, 0, date('m'), date('d'), date('Y') + $ptkr_time);
+} elseif (substr($GLOBALS['ptkr_end_date'], 0, 1) == 'M') {
+    $ptkr_time = substr($GLOBALS['ptkr_end_date'], 1, 1);
+    $ptkr_future_time = mktime(0, 0, 0, date('m') + $ptkr_time, date('d'), date('Y'));
+} elseif (substr($GLOBALS['ptkr_end_date'], 0, 1) == 'D') {
+    $ptkr_time = substr($GLOBALS['ptkr_end_date'], 1, 1);
+    $ptkr_future_time = mktime(0, 0, 0, date('m'), date('d') + $ptkr_time, date('Y'));
 }
 
+$to_date = date('Y-m-d', $ptkr_future_time);
+$to_date = !is_null($_REQUEST['datepicker2']) ? date('Y-m-d', strtotime($_REQUEST['datepicker2'])) : $to_date;
 $form_patient_name = !is_null($_POST['form_patient_name']) ? $_POST['form_patient_name'] : null;
 $form_patient_id = !is_null($_POST['form_patient_id']) ? $_POST['form_patient_id'] : null;
 
+// get all appts for date range and refine view client side.  very fast...
 $appointments = array();
-$from_date = date("Y-m-d");
-$to_date = date("Y-m-d");
 $datetime = date("Y-m-d H:i:s");
-
-# go get the information and process it
-$appointments = fetch_Patient_Tracker_Events($from_date, $to_date, $provider, $facility, $form_apptstatus, $form_apptcat, $form_patient_name, $form_patient_id);
-$appointments = sortAppointments($appointments, 'time');
-
+$appointments = fetch_Patient_Tracker_Events($from_date, $to_date, '', '', '', '', $form_patient_name, $form_patient_id);
```

Comment:
TODO
Look into this and ensure no features have been lost (I am guessing another mechanism is being used to do this now, so really just a note for myself to remember)